test/Microsoft.Sbom.Targets.Tests/AbstractGenerateSBomTaskInputTests.cs

```diff
@@ -226,9 +256,11 @@ public void Sbom_Generation_Succeeds_For_Null_Verbosity()
     public void Sbom_Generation_Succeeds_For_Invalid_Verbosity()
     {
         // Arrange
-        // If an invalid Verbosity is specified, the default value should be Verbose and is printed in the
-        // tool's standard output.
+        // If an invalid Verbosity is specified, the default value should be Verbose. It is also printed in the
+        // tool's standard output for the MSBuild Core task.
+#if !NET472
```

Comment:
I think we can use this pattern matching for both GenerateSbomTask and SbomCLIToolTask. I believe both of them output the same JSON string with all the properties, including the Verbosity

Reply:
Thank you Varshita. Addressed at 770707deabb8f34a99e00cdfc2fff87e696ea25a

Reply:
Also created this issue: https://github.com/microsoft/sbom-tool/issues/616